src/Components/DayForm.js

```diff
@@ -30,28 +48,22 @@ export default function DayForm({ onSubmit, day }) {
           onSubmit={handleSubmit}
         >
           <StyledDay>{day}</StyledDay>
-          {/* <SpaceBetween>
-          <div>
-            <p>mucels</p>
-          </div>
-          <StyledRow>
-            <button type="button">
-              <IoIosAddCircleOutline />
-            </button>
-            <p>0</p>
-            <Button name="-" />
-          </StyledRow>
-        </SpaceBetween> */}
           <SpaceBetween>
```

Comment:
What kind of component is SpaceBetween? Maybe you could be more descriptive? :)

Reply:
this is just a StyledComponent, this will make a space between 2 elements :)